### PR TITLE
Implement ros provider

### PR DESCRIPTION
**Pull request by Prodesire**

### CODE REVIEW

File: alibabacloud/ros/Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+TAG ?= latest
+
+PROJECT_NAME := "ros-oam"
+PKG := "gitlab.alibaba-inc.com/ros/$(PROJECT_NAME)"
```
Comment (hongchaodeng) [inline alibabacloud/ros/Makefile]: Need to change PKG name

Comment (Prodesire) [reply alibabacloud/ros/Makefile]: Changed it. Thanks.

**Comment by hongchaodeng** [comment]
lgtm

**Review by wonderflow — APPROVED** [review]
LGTM

**Pull request merged**